src/main/kotlin/ee/carlrobert/codegpt/codecompletions/InfillPromptTemplate.kt

```diff
@@ -12,6 +12,11 @@ enum class InfillPromptTemplate(val label: String, val stopTokens: List<String>?
             return "<PRE> $prefix <SUF>$suffix <MID>"
         }
     },
+    LLAMA_3("Llama 3", listOf("<|eot_id|>")) {
+        override fun buildPrompt(prefix: String, suffix: String): String {
+            return "<|start_header_id|>user<|end_header_id|>\n\n$prefix\n$suffix<|eot_id|><|start_header_id|>assistant<|end_header_id|>\n\n"
```

Comment:
That is just the general chat prompt template for LLaMa 3 but for actual code completions we should use FIM (fill in the middle), which is not supported by any base LLama 2 / 3 model.
Should probably wait for a Codellama or similar version based on Llama 3

Reply:
I'll remove it then, thank you!